classes/music_album.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify: false)
+    super()
+    @on_spotify = on_spotify
+  end
+
+  private
+
+  def can_be_archived?
+    current_year = Time.new.year
+    current_year - @publish_date > 10 && @on_spotify == true
```

Comment:
Here you don't have to repeat yourself again you just need to call super keyword to call parent's method I suggest that to prevent (DRY)